src/sqlfluff/core/parser/segments/base.py

```diff
@@ -1053,7 +1053,6 @@ def apply_fixes(self, dialect, rule_code, fixes):
                                 )
                             # We've applied a fix here. Move on, this also consumes the
                             # fix
-                            # TODO: Maybe deal with overlapping fixes later.
```

Comment:
Delete this comment now that we're dealing with them (by warning and discarding)